CMakeLists.txt

```diff
@@ -7,7 +7,11 @@ add_subdirectory(lib/open-simulation-interface)
 get_directory_property(OSI_VERSION_MAJOR DIRECTORY lib/open-simulation-interface DEFINITION VERSION_MAJOR)
 get_directory_property(OSI_VERSION_MINOR DIRECTORY lib/open-simulation-interface DEFINITION VERSION_MINOR)
 get_directory_property(OSI_VERSION_PATCH DIRECTORY lib/open-simulation-interface DEFINITION VERSION_PATCH)
-set(OSIVERSION "${OSI_VERSION_MAJOR}.${OSI_VERSION_MINOR}.${OSI_VERSION_PATCH}")
+# set(OSIVERSION "${OSI_VERSION_MAJOR}.${OSI_VERSION_MINOR}.${OSI_VERSION_PATCH}")
+# CMOSI bugfix, CMOSI interface expects OSI 3.0.0, although supporting 3.5.0
+set(OSIVERSION "3.0.0")
+set(OSMPVERSION "1.0.0" CACHE STRING "OSMP Version String")
+set(MODELVERSION "0.0.1" CACHE STRING "Model Version String")
```

Comment:
Since we don't have a release for the template, yet, we can leave it as 0.0.1. We just have to remember to increase it, once we make a release.

Reply:
Besides that, the main reason was to test that Carmaker is not checking this line for the 1.0.0 OSMP version